src/sql/mod.rs

```diff
@@ -21,25 +21,34 @@ pub(crate) enum SqlBuilder {
 pub(crate) enum Part {
     Arg,
     Fields,
+    Str(&'static str),
     Text(String),
 }
 
 impl SqlBuilder {
     pub(crate) fn new(template: &str) -> Self {
```

Comment:
with removal of lifetime generics, a constructor could still be added for when query is `&'static str`

Anyways, that'd be a different PR